script/models/playbackstrategy.js

```diff
@@ -0,0 +1,12 @@
+define(
+    'bigscreenplayer/models/playbackstrategy', [],
```

Comment:
I'd argue that this shouldn't exist. The abstraction leaks throughout the codebase. I know a similar thing is done in hybridstrategy, but that also shouldn't exist as a module to be honest.

Reply:
With this piece of work, we will now need to determine whether we are in `talstrategy` or not for the failover logic. This means we will need to use these strings in three files: `mediaresilience`, `strategypicker` & `hybridstrategy`. Do you think we should keep using the plain strings?